addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/handler/OnkyoHandler.java

```diff
@@ -816,4 +819,15 @@ public PercentType getVolume() throws IOException {
     public void setVolume(PercentType volume) throws IOException {
         handleVolumeSet(EiscpCommand.Zone.ZONE1, volumeLevelZone1, downScaleVolume(volume));
     }
+
+    private String guessMimeTypeFromData(byte[] data) {
+        String mimeType = RawType.DEFAULT_MIME_TYPE;
```

Comment:
This is not needed, it will always be overridden on the next line.

Reply:
Indeed, previous guess function did throw exception